seed/dns.go

```diff
@@ -96,8 +99,8 @@ func (ds *DnsServer) locateChainView(subdomain string) *ChainView {
 
 	subdomain = strings.TrimSpace(subdomain)
 	segments := strings.SplitAfter(subdomain, ".")
-	//	log.Debug("seg: ", segments)
-	//log.Debug("seg: ", len(segments))
+	log.Debug("seg: ", segments)
```

Comment:
Was this accidental? These were some stray debugging logs, that can likely be removed.

Reply:
Removed.
Needed the debug output initially to see what's going on.

Reply:
removed